packages/react/src/Dialog.ts

```diff
@@ -0,0 +1,2 @@
+export * from 'office-ui-fabric-react/lib/Dialog';
+export { Dialog as default } from 'office-ui-fabric-react/lib/Dialog';
```

Comment:
What is this doing here?

Reply:
Honestly not sure why we have a default export here.